redash/query_runner/druid.py

```diff
@@ -48,7 +55,9 @@ def run_query(self, query, user):
         connection = connect(host=self.configuration['host'],
                              port=self.configuration['port'],
                              path='/druid/v2/sql/',
-                             scheme=self.configuration['scheme'])
+                             scheme=self.configuration['scheme'],
```

Comment:
```suggestion
                             scheme=self.configuration.get('scheme', 'http'),
```

(this was an issue before your PR, but worth addressing)